### DISCUSSION TITLE
Relationship not rendering in included file

### DISCUSSION THREAD

**Discussion by TonyK-FmGroup** [structurizr/lite#172] [Q&A]
Hi all,
I'm using the docker image of Lite and we have a diagram where we've put a lot of relationships in a separate file and then include that file. The file appears to be read because if i try to create an invalid relationship then I see an error.  However, these relationships in this file do not render in the diagram when I just have include *  in the deployment view.
Have I done something wrong here?

Thanks in advance!

**Comment by simonbrowndotje** [comment]
I suspect what you're seeing is unrelated to the relationships being defined in a separate file. It's hard to help further without a reproducible example though.
